lib/ui/views/login_view.dart

```diff
@@ -8,25 +8,38 @@ import 'package:provider_start/ui/shared/ui_helper.dart';
 import 'package:provider_start/ui/widgets/loading_animation.dart';
 
 class LoginView extends StatelessWidget {
+  final _formKey = GlobalKey<FormState>();
+  final _emailController = TextEditingController();
+  final _passwordController = TextEditingController();
+  final _passwordFocusNode = FocusNode();
+
   @override
   Widget build(BuildContext context) {
     final local = AppLocalizations.of(context);
 
     return ViewModelProvider<LoginViewModel>.withoutConsumer(
       viewModel: LoginViewModel(),
       builder: (context, model, child) => GestureDetector(
-        onTap: () => FocusScope.of(context).requestFocus(model.viewFocusNode),
+        onTap: () => FocusScope.of(context).requestFocus(FocusNode()),
```

Comment:
I think this focusNode should be created once and added to the dispose method of login view instead. As to not keep rebuilding the FocusNode and causing potential memory leaks.

Reply:
I saw that code like that somewhere, I'm not sure now the place but maybe you are right. I think the new FocusNode() is in the context of requestFocus, so that's why there is no problem with that, but I can't confirm it yet.

Reply:
Let's just be safe and add a viewFocusNode

Reply:
Code refactor